hw/mips/malta.c

```diff
@@ -70,6 +71,13 @@
 #define FPGA_ADDRESS        0x1f000000ULL
 #define RESET_ADDRESS       0x1fc00000ULL
 
+// Maximum number of virtio transports. These sit idle if no devices are specified when QEMU is run.
+#define VIRTIO_N_TRANSPORTS    2
+
+#define VIRTIO_MMIO_MMAP_BASE     0x1e400000ULL
+#define VIRTIO_MMIO_MMAP_SIZE     0x200ULL
```

Comment:
RISC-V's virt machine allocates 4K per device, though ARM's allocates 512B as you've done so this is probably fine.

Reply:
I think 512B is sensible on a CHERI architecture because we can bound more precisely than a page.

Reply:
Who cares if you can wander off into more unmapped memory? Even for 512B there will be a portion of it that's unused. I don't think it matters either way.